Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIScriptingCocoa.mm

```diff
@@ -192,9 +235,9 @@
         return;
 
     Vector<WebExtensionRegisteredScriptParameters> parameters;
-    parseRegisteredContentScripts(scripts, parameters);
+    parseRegisteredContentScripts(scripts, true, parameters);
```

Comment:
An `enum class : bool { No, Yes }` for this would help make the call site clear what true / false means.